src/rust/ensogl/lib/text/src/component/area.rs

```diff
@@ -405,6 +405,8 @@ ensogl_core::define_endpoints! {
         redo(),
         /// Copy selected text to clipboard.
         copy(),
+        /// Copy selected text to clipboard and remove from area.
```

Comment:
The sentence is not 100% english correct. Change it to:
"Copy the selected text to the clipboard and remove it from the text area."

Fix the above doc as well please